src/lib/common.js

```diff
@@ -34,6 +34,21 @@ const common = {
     return pixels * ProjectConstants.MS_PER_PIXEL;
   },
 
+  gridTimesAround: function (from, to, {numMeasures, timeInterval}) {
```

Comment:
gridTimesAround, what does that mean? This should be renamed and commented. also this is only used in one file, should it be in common?